tests/console/zypper_docker.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test zypper-docker installation and its usage
+#    Cover the following aspects of zypper-docker:
+#      * zypper-docker package can be installed
+#      * zypper-docker can list local images:                    'zypper-docker images ls'
+#      * zypper-docker can list updates/patches:                 'zypper-docker list-updates' 'zypper-docker list-patches'
+#      * zypper-docker can list outdated containers:             'zypper-docker ps'
+#      * zypper-docker can list updates/patches for a container: 'zypper-docker list-updates-container' 'zypper-docker list-patches-container'
+#      * zypper-docker can apply the updates:                    'zypper-docker update'
+# Maintainer: Antonio Caristia <acaristia@suse.com>
+
+use base "consoletest";
+use testapi;
+use utils;
+use strict;
+use registration;
+use version_utils qw(is_sle is_leap);
```

Comment:
you don't seem to use this, please delete the above two lines

Reply:
utils is for zypper_call() , registration for install_docker_when_needed()
But,you are right, the other ones are useless!